### ISSUE TITLE
Have a vulnerable AI project :-)?

### ISSUE THREAD

**Issue by commjoen** [OWASP/www-project-ai-security-and-privacy-guide#6]
Hi there!
This is a great project! Thank you for documenting it so nicely!
To make it more practical: will there ever be a “vulnerable AI” project where we can make some of the attacks described more practical :-)?

**Comment by robvanderveer** [comment]
Robert Sullivan is writing a few articles/tutorials with code to practice attacks. https://www.linkedin.com/pulse/five-security-risks-every-machine-learning-engineer-must-sullivan/

**Comment by robvanderveer** [comment]
And thanks. Let's discuss this idea on slack or in a call Jeroen.

**Comment by mspsullivan** [comment]
True. Your interest is well-timed. I see a need for a project that can
demonstrate top vulnerabilities,  then train defenders to block or detect
them.
I have two examples (#2 to be published soon) and a third in the works.
For now, I suggest we share examples we have seen & had sucess running.
I have looked at Adversarial Robustness Toolbox (ART ), there is also
Counterfit from Microsoft.

Robert.

**Comment by mchechik** [comment]
though less well-timed than the OP, WithSecure Labs recently published the [damn-vulnerable-llm-agent](https://github.com/WithSecureLabs/damn-vulnerable-llm-agent) - a vulnerable chatbot (see project readme for more info)
